ocd-3016.sql

```diff
@@ -0,0 +1,16 @@
+do $$
+begin
+    if exists(select count(*) from openchpl.activity_concept where concept = 'COMPLAINT') THEN
+        -- Just set the seq to the correct value
+        perform setval('openchpl.activity_concept_activity_concept_id_seq', 16);
```

Comment:
This seems like it will work..not necessarily requesting a change, but would it be simpler to set the sequence to the count(*) value of the activity_concept table?